### PR TITLE
Added alternative setup using wstool and a rosinstall file

### PR DESCRIPTION
**Pull request by ipa-nhg**
I have added a rosinstall file with the non-released repos and add some instructions to install the dependencies automatically, this ensures that we all installed the same set of ROS packages.

:exclamation::exclamation::exclamation::exclamation:
I added a "-r" argument to the rosdep command (i.e. *Continue installing despite errors*) because of some abb libraries that can't be found ** . that means the rosdep command will show the following errors:
```
yumi_hw: Cannot locate rosdep definition for [abb_rws_interface]
yumi_cameras: Cannot locate rosdep definition for [abb_rws_interface]
```
Both errors can be ignored because for the CMakeLists the abb_rws_interface package is marked as additional library and the compile command will not fail.

** for further information: https://answers.ros.org/question/265655/unable-build-abb-yumi-support/
:exclamation::exclamation::exclamation::exclamation:

### CODE REVIEW

File: metacontrol_sim.rosinstall [hunk 1]
```diff
@@ -0,0 +1,12 @@
+- git:
+    local-name: metacontrol_sim
+    uri: https://github.com/rosin-project/metacontrol_sim
+- git:
+    local-name: yumi
+    uri: https://github.com/marioney/yumi
+- git:
+    local-name: metacontrol_nav
+    uri: https://github.com/marioney/metacontrol_nav
```
Comment (marioney) [inline metacontrol_sim.rosinstall:9]: The metacontrol_nav package has been moved to the rosin-project.  The correct address here should be

https://github.com/rosin-project/metacontrol_nav

**Comment by marioney** [comment]
Thanks a lot for this, It's very useful!  I've sent you a comment regarding the address of the metacontrol_nav package, that is now in the rosin-project https://github.com/rosin-project/metacontrol_nav

**Comment by marioney** [comment]
>  that is now in the rosin-project https://github.com/rosin-project/metacontrol_nav

I've just changed this on the README file.

File: metacontrol_sim.rosinstall (hunk 1 above)
Comment (ipa-nhg) [reply metacontrol_sim.rosinstall]: Change done and history rebased! Now should be fine :+1:

**Pull request merged**